census/forms.py

```diff
@@ -14,6 +14,8 @@ class Meta:
             'event_type',
```

Comment:
We need 'site_name' as well as location, to match the spread sheets that Ashley gave us.